packages/node_modules/@webex/internal-plugin-conversation/test/integration/spec/verbs.js

```diff
@@ -804,6 +805,43 @@ describe('plugin-conversation', function () {
         });
       });
 
+      describe('#react', () => {
```

Comment:
Can you split this into two suites? `describe('#deleteReaction()')` and `describe('#addReaction()')`